api/src/main/java/org/apache/cloudstack/api/command/user/address/DisassociateIPAddrCmd.java

```diff
@@ -46,10 +46,13 @@ public class DisassociateIPAddrCmd extends BaseAsyncCmd {
     //////////////// API parameters /////////////////////
     /////////////////////////////////////////////////////
 
-    @Parameter(name = ApiConstants.ID, type = CommandType.UUID, entityType = IPAddressResponse.class, required = true, description = "the ID of the public IP address"
+    @Parameter(name = ApiConstants.ID, type = CommandType.UUID, entityType = IPAddressResponse.class, description = "the ID of the public IP address"
         + " to disassociate")
     private Long id;
 
+   @Parameter(name=ApiConstants.IP_ADDRESS, type=CommandType.STRING, description="IP Address to be disassociated")
```

Comment:
```suggestion
   @Parameter(name=ApiConstants.IP_ADDRESS, type=CommandType.STRING, description="IP Address to be disassociated", since="4.19.0")
```

Would be good to add since property

Reply:
Will add it in the next commit.